src/main/kotlin/no/nav/familie/ef/sak/infrastruktur/featuretoggle/FeatureToggleConfig.kt

```diff
@@ -1,25 +1,14 @@
 package no.nav.familie.ef.sak.infrastruktur.featuretoggle
 
 import io.getunleash.strategy.Strategy
-import no.nav.familie.unleash.DefaultUnleashService
-import org.springframework.beans.factory.annotation.Value
 import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 
 @Configuration
-class FeatureToggleConfig(
-    @Value("\${UNLEASH_SERVER_API_URL}") private val apiUrl: String,
-    @Value("\${UNLEASH_SERVER_API_TOKEN}") private val apiToken: String,
-    @Value("\${NAIS_APP_NAME}") private val appName: String,
-) {
+class FeatureToggleConfig {
```

Comment:
Kan hele denne slettes?
Evt. flytte strategies til `UnleashConfig` ?

Reply:
UnleashConfig ligger i familie, og vi må bønne opp strategies selv

Reply:
Godt poeng. Kan vi rename filen til UnleashCustomConfig eller noe? Så man skjønner at det "henger sammen" ?

Reply:
Mulig det funker, men litt finurlig å ha en bønne som heter `strategies` som skal injectes i et eksternt bibliotek. 
Burde kanskje hete `CustomUnleashStrategies` eller noe i den duren - for eksplisitt å forstå at _DETTE_ blir injecta i noe annet?

Reply:
Ja, endrer til CustomUnleashStrategies